### PR TITLE
Add MCNC 2019 article

### PR DESCRIPTION
**Pull request by xz1z1z1z1z**
add MCNC tournament to the wiki page

### CODE REVIEW

File: wiki/Tournaments/MCNC/2019/en.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+  - [Camellia - NEURO-CLOUD-9 (Evening) [GAUSSIAN (EVE.NING'S FLIP) x1.05]](https://osu.ppy.sh/b/1884094)
+- Tiebreaker
+  - **[Unlucky Morpheus - Sono Tamashii Ni Yasuragi Wo ~ Dignity of Spirit (Evening) [forever and never]](https://osu.ppy.sh/b/2095156)**
+
+### Semifinals
+
+**[Download the mappack here! (96.5MB)](https://www.dropbox.com/s/fq0u3l87fahgw5b/MCNC19%20%E5%8D%8A%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [Alfheim (Tidek) [Freya]](https://osu.ppy.sh/b/1966772)
```
Comment (Nivalyx) [inline wiki/Tournaments/MCNC/2019/en.md:133]: this map should be listed as

> [Kakka - Alfheim (Tidek) [Freya]](https://osu.ppy.sh/b/1966772)

File: wiki/Tournaments/MCNC/2019/en.md [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+- Freemod
+  - [TAG underground overlay - Over The "Period" (Tidek) [Edit]](https://osu.ppy.sh/b/1777377)
+  - [lapix - Carry me away (Couil) [4K SV CHALLENGE]](https://osu.ppy.sh/b/1271753)
+  - [sky_delta - Precursor (Daughter-) [antecedent]](https://osu.ppy.sh/b/1791811)
+  - [ck - Carnation (ck remix) (Gekido-) [LN Master]](https://osu.ppy.sh/b/1579095)
+  - [Camelia - The King Of Lions ([OSC]Agami) [Jungle Warfare]](https://osu.ppy.sh/b/2094458)
+  - [Inferi - Those Who From the Heavens Came (XeoStyle) [Investiture of the Gods]](https://osu.ppy.sh/b/897408)
+  - [DJKurara - Steady Rock Kiss On Me (drunkenstein) [Happiness]](https://osu.ppy.sh/b/1682118)
+  - [a_hisa - Smoked Turkey Rag ([Illustions]) [tournament?]](https://osu.ppy.sh/b/2096435)
+  - [aaaa vs Frums - beepbit *** futures (LeiN-) [cosmonaut]](https://osu.ppy.sh/b/1864667)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:118]:
```suggestion
  - [aaaa vs Frums - beepbit \*\*\* futures (LeiN-) [cosmonaut]](https://osu.ppy.sh/b/1864667)
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 3]
```diff
@@ -0,0 +1,354 @@
...
+
+- Freemod
+  - [TAG underground overlay - Over The "Period" (Tidek) [Edit]](https://osu.ppy.sh/b/1777377)
+  - [lapix - Carry me away (Couil) [4K SV CHALLENGE]](https://osu.ppy.sh/b/1271753)
+  - [sky_delta - Precursor (Daughter-) [antecedent]](https://osu.ppy.sh/b/1791811)
+  - [ck - Carnation (ck remix) (Gekido-) [LN Master]](https://osu.ppy.sh/b/1579095)
+  - [Camelia - The King Of Lions ([OSC]Agami) [Jungle Warfare]](https://osu.ppy.sh/b/2094458)
+  - [Inferi - Those Who From the Heavens Came (XeoStyle) [Investiture of the Gods]](https://osu.ppy.sh/b/897408)
+  - [DJKurara - Steady Rock Kiss On Me (drunkenstein) [Happiness]](https://osu.ppy.sh/b/1682118)
+  - [a_hisa - Smoked Turkey Rag ([Illustions]) [tournament?]](https://osu.ppy.sh/b/2096435)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:117]:
```suggestion
  - [a\_hisa - Smoked Turkey Rag ([Illustions]) [tournament?]](https://osu.ppy.sh/b/2096435)
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 4]
```diff
@@ -0,0 +1,354 @@
...
+  - [aaaa vs Frums - beepbit *** futures (LeiN-) [cosmonaut]](https://osu.ppy.sh/b/1864667)
+  - [Leaf - LeaF  Style Super Shredder II (Imperial Wolf) [Shred everything]](https://osu.ppy.sh/b/2100461)
+  - [Lamberti phil - Alpha ([Crz]Zetsfy) [Reincarnation]](https://osu.ppy.sh/b/2046040)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:120]:
```suggestion
  - [Lamberti phil - Alpha (\[Crz\]Zetsfy) [Reincarnation]](https://osu.ppy.sh/b/2046040)
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 5]
```diff
@@ -0,0 +1,354 @@
...
+  - [KopophobiA & Ajataa - Lunar Eclipse ([Crz]Zetsfy) [Firanzia]](https://osu.ppy.sh/b/1900586)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:121]:
```suggestion
  - [KopophobiA & Ajataa - Lunar Eclipse (\[Crz\]Zetsfy) [Firanzia]](https://osu.ppy.sh/b/1900586)
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 6]
```diff
@@ -0,0 +1,354 @@
...
+---------------------------
+
+## Mappools
+
+### Finals
+
+**[Download the mappack here! (154MB)](https://www.dropbox.com/s/r4p7tbxa58ijgzh/MCNC19%20%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [TAG underground overlay - Over The "Period" (Tidek) [Edit]](https://osu.ppy.sh/b/1777377)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:110]:
```suggestion
  - [TAG underground overlay - Over The "Period" (Tidek) \[Edit\]](https://osu.ppy.sh/beatmaps/1777377)
```
Use latest linking. Apply for all /b/
And escape all brackets used to wrap the difficulty name `\[Edit\]`. Apply for all too.

File: wiki/Tournaments/MCNC/2019/en.md [hunk 7]
```diff
@@ -0,0 +1,354 @@
...
+  - [ck - Carnation (ck remix) (Gekido-) [LN Master]](https://osu.ppy.sh/b/1579095)
+  - [Camelia - The King Of Lions ([OSC]Agami) [Jungle Warfare]](https://osu.ppy.sh/b/2094458)
+  - [Inferi - Those Who From the Heavens Came (XeoStyle) [Investiture of the Gods]](https://osu.ppy.sh/b/897408)
+  - [DJKurara - Steady Rock Kiss On Me (drunkenstein) [Happiness]](https://osu.ppy.sh/b/1682118)
+  - [a_hisa - Smoked Turkey Rag ([Illustions]) [tournament?]](https://osu.ppy.sh/b/2096435)
+  - [aaaa vs Frums - beepbit *** futures (LeiN-) [cosmonaut]](https://osu.ppy.sh/b/1864667)
+  - [Leaf - LeaF  Style Super Shredder II (Imperial Wolf) [Shred everything]](https://osu.ppy.sh/b/2100461)
+  - [Lamberti phil - Alpha ([Crz]Zetsfy) [Reincarnation]](https://osu.ppy.sh/b/2046040)
+  - [KopophobiA & Ajataa - Lunar Eclipse ([Crz]Zetsfy) [Firanzia]](https://osu.ppy.sh/b/1900586)
+  - [orangentle - HAELEQUIN -the faith of truth- (_underjoy) [Shattered Personality]](https://osu.ppy.sh/b/1512722)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:122]:
```suggestion
  - [orangentle - HAELEQUIN -the faith of truth- (\_underjoy) [Shattered Personality]](https://osu.ppy.sh/b/1512722)
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 8]
```diff
@@ -0,0 +1,354 @@
...
+  - [Quarks (kradness&Camellia) - Intro + Dualive (Evening) [backpropogation]](https://osu.ppy.sh/b/2084873)
+  - [Camellia - NEURO-CLOUD-9 (Evening) [GAUSSIAN (EVE.NING'S FLIP) x1.05]](https://osu.ppy.sh/b/1884094)
+- Tiebreaker
+  - **[Unlucky Morpheus - Sono Tamashii Ni Yasuragi Wo ~ Dignity of Spirit (Evening) [forever and never]](https://osu.ppy.sh/b/2095156)**
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:126]:
```suggestion
  - **[Unlucky Morpheus - Sono Tamashii Ni Yasuragi Wo \~ Dignity of Spirit (Evening) [forever and never]](https://osu.ppy.sh/b/2095156)**
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 9]
```diff
@@ -0,0 +1,354 @@
...
+
+**[Download the mappack here! (82.9MB)](https://www.dropbox.com/s/k752wba6kpyyzt1/MCNC19%20RO16%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- ModType
+  - [Frums - Wavetapper (qqqant) [Stage 1: Memory Factory]](https://osu.ppy.sh/b/1811863)
+  - [stereoberry - evangelize (blurry images) (Tidek) [transparency (SV)]](https://osu.ppy.sh/b/1222765)
+  - [Beltaine - Rockhill (Ryu Sei) [Hard]](https://osu.ppy.sh/b/994291)
+  - [PROTODOME - Greatest Hat (Gekido-) [chalLeNge]](https://osu.ppy.sh/b/1759313)
+  - [Frederic - oddloop (infilect) [oldnep]](https://osu.ppy.sh/b/1376904)
+  - [eye burn - SHI*A*WA*SE (Elekton) [joy]](https://osu.ppy.sh/b/1552444)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:182]:
```suggestion
  - [eye burn - SHI\*A\*WA\*SE (Elekton) [joy]](https://osu.ppy.sh/b/1552444)
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 10]
```diff
@@ -0,0 +1,354 @@
...
+  - [wa. remixed by celas - Gin no Kaze (Elekton) [mistral]](https://osu.ppy.sh/b/1592941)
+  - [Silentroom - Protoflicker ([Crz]Crysarlene) [crysather]](https://osu.ppy.sh/b/1954837)
+  - [Aural Vampire - Shonan Zoku -Cannibal Coast- (Wh1teh) [k]](https://osu.ppy.sh/b/1158510)
+  - [The Flashbulb - Chik Habit (Gekido-) [Stage 4: Torment]](https://osu.ppy.sh/b/1546411)
+  - [Laur, Dustvoxx - FireLight (Neokontrol Remix) (Suu is my waifu) [Enkindle]](https://osu.ppy.sh/b/1708743)
+  - [Soko ni Naru - Tenohira de Odoru (Vortex-) [Sweaty Palms]](https://osu.ppy.sh/b/1711260)
+  - [Camellia - *Feels Seasickness...* (Suu is my waifu) [*Maximum...*]](https://osu.ppy.sh/b/1932640)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:189]:
```suggestion
  - [Camellia - \*Feels Seasickness...\* (Suu is my waifu) [\*Maximum...\*]](https://osu.ppy.sh/b/1932640)
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 11]
```diff
@@ -0,0 +1,354 @@
...
+  - [4k - SHD+ Package (1234567890) [Tuk Tuk Boshi (Jump Hand+)]](https://osu.ppy.sh/b/838065)
+  - [SHK.o2SE - Death Moon (Guilhermeziat) [Dawn of The Final Day]](https://osu.ppy.sh/b/1412004)
+  - [Terminal 11 - That's Gonna Be (Gekido-) [Challenge]](https://osu.ppy.sh/b/2051696)
+  - [The Ghost of 3.13 - Forgotten (Shoegazer) [Forgotten Collab]](https://osu.ppy.sh/b/749258)
+  - [-45 - Utan2 (976600146) [-]](https://osu.ppy.sh/b/1556704)
+  - [Chroma - I (Guilhermeziat) [You]](https://osu.ppy.sh/b/1592359)
+  - [gmtn. (witch's slave) - Daraku no Sono (shadowsnoop) [Tumultu]](https://osu.ppy.sh/b/1132175)
+  - [Sullivan King & Matt McGuire - Lockdown (Feat.Sam King) (Song Hana) [Declear an Emergency]](https://osu.ppy.sh/b/1247641)
+- Tiebreaker
+  - **[aaaa - Hoshi wo Kakeru Adventure ~ we are forever friends! ~ [Long ver.] (Daikyi) [Our Epilogue]](https://osu.ppy.sh/b/1074892)**
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:149]:
```suggestion
  - **[aaaa - Hoshi wo Kakeru Adventure \~ we are forever friends! \~ \[Long ver.\] (Daikyi) [Our Epilogue]](https://osu.ppy.sh/b/1074892)**
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 12]
```diff
@@ -0,0 +1,354 @@
...
+
+### Semifinals
+
+**[Download the mappack here! (96.5MB)](https://www.dropbox.com/s/fq0u3l87fahgw5b/MCNC19%20%E5%8D%8A%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [Kakka - Alfheim (Tidek) [Freya]](https://osu.ppy.sh/b/1966772)
+  - [lapix - Foolish Hero (dionzz99) [Fooled]](https://osu.ppy.sh/b/1040762)
+  - [U1 overground - Dopamine (juankristal) [Polyethylene Oxide]](https://osu.ppy.sh/b/1472944)
+  - [SOUND HOLIC - WANNA BE CRAZY PRINCESS -  ([OSC]Agami) [/L/unatic Pri/N/cess]](https://osu.ppy.sh/b/1860579)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:136]:
```suggestion
  - [SOUND HOLIC - WANNA BE CRAZY PRINCESS -  (\[OSC\]Agami) [/L/unatic Pri/N/cess]](https://osu.ppy.sh/b/1860579)
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 13]
```diff
@@ -0,0 +1,354 @@
...
+  - [Sullivan King & Matt McGuire - Lockdown (Feat.Sam King) (Song Hana) [Declear an Emergency]](https://osu.ppy.sh/b/1247641)
+- Tiebreaker
+  - **[aaaa - Hoshi wo Kakeru Adventure ~ we are forever friends! ~ [Long ver.] (Daikyi) [Our Epilogue]](https://osu.ppy.sh/b/1074892)**
+
+### Quarterfinals
+
+**[Download the mappack here! (110MB)](https://www.dropbox.com/s/6ocxk2bm9w8q49j/MCNC19%20%E5%9B%9B%E5%88%86%E4%B9%8B%E4%B8%80%E5%86%B3%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- ModType
+  - [Jameson Hodge - pn sqnc 518[6] (Couil) [sv sqnc 518[1]]](https://osu.ppy.sh/b/2037752)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:156]:
```suggestion
  - [Jameson Hodge - pn sqnc 518\[6\] (Couil) [sv sqnc 518[1]]](https://osu.ppy.sh/b/2037752)
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 14]
```diff
@@ -0,0 +1,354 @@
...
+
+- ModType
+  - [Frums - Wavetapper (qqqant) [Stage 1: Memory Factory]](https://osu.ppy.sh/b/1811863)
+  - [stereoberry - evangelize (blurry images) (Tidek) [transparency (SV)]](https://osu.ppy.sh/b/1222765)
+  - [Beltaine - Rockhill (Ryu Sei) [Hard]](https://osu.ppy.sh/b/994291)
+  - [PROTODOME - Greatest Hat (Gekido-) [chalLeNge]](https://osu.ppy.sh/b/1759313)
+  - [Frederic - oddloop (infilect) [oldnep]](https://osu.ppy.sh/b/1376904)
+  - [eye burn - SHI*A*WA*SE (Elekton) [joy]](https://osu.ppy.sh/b/1552444)
+  - [wa. remixed by celas - Gin no Kaze (Elekton) [mistral]](https://osu.ppy.sh/b/1592941)
+  - [Silentroom - Protoflicker ([Crz]Crysarlene) [crysather]](https://osu.ppy.sh/b/1954837)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:184]:
```suggestion
  - [Silentroom - Protoflicker (\[Crz\]Crysarlene) [crysather]](https://osu.ppy.sh/b/1954837)
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 15]
```diff
@@ -0,0 +1,354 @@
...
+  - [Aural Vampire - Shonan Zoku -Cannibal Coast- (Wh1teh) [k]](https://osu.ppy.sh/b/1158510)
+  - [The Flashbulb - Chik Habit (Gekido-) [Stage 4: Torment]](https://osu.ppy.sh/b/1546411)
+  - [Laur, Dustvoxx - FireLight (Neokontrol Remix) (Suu is my waifu) [Enkindle]](https://osu.ppy.sh/b/1708743)
+  - [Soko ni Naru - Tenohira de Odoru (Vortex-) [Sweaty Palms]](https://osu.ppy.sh/b/1711260)
+  - [Camellia - *Feels Seasickness...* (Suu is my waifu) [*Maximum...*]](https://osu.ppy.sh/b/1932640)
+- Tiebreaker
+  - **[goreshit - fleshbound ([Crz]NucleoAP) [Couil's trapped]](https://osu.ppy.sh/b/2075130)**
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:191]:
```suggestion
  - **[goreshit - fleshbound (\[Crz\]NucleoAP) [Couil's trapped]](https://osu.ppy.sh/b/2075130)**
```
Escaping

File: wiki/Tournaments/MCNC/2019/en.md [hunk 16]
```diff
@@ -0,0 +1,354 @@
...
+
+### Group Stage
+
+**[Download the mappack here! (84.3MB)](https://www.dropbox.com/s/ywwvekemymwzrjh/MCNC19%20%E5%B0%8F%E7%BB%84%E8%B5%9B%E5%9B%BE%E5%8C%85.rar?dl=0)**
+
+- Freemod
+  - [Frums - Credits (BossModWolf) [Glitched Tech Corp. (SV)]](https://osu.ppy.sh/b/1985858)
+  - [succducc - me & u (Couil) [& sv]](https://osu.ppy.sh/b/1442650)
+  - [FELT - Life [Eris's "a Fate Encounter" mix] ([OSC]Amagi) [Lunatic]](https://osu.ppy.sh/b/1971029)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:200]:
```suggestion
  - [FELT - Life [Eris's "a Fate Encounter" mix] (\[OSC\]Amagi) [Lunatic]](https://osu.ppy.sh/b/1971029)
```

File: wiki/Tournaments/MCNC/2019/en.md [hunk 17]
```diff
@@ -0,0 +1,354 @@
...
+  - [Draw the Emotional & Foreground Eclipse - Stay by my side (Herkkupala) [puxtu's Lunatic]](https://osu.ppy.sh/b/968811)
+  - [ARM - Saita, Kimi no Egao (Julie) [I can Fly]](https://osu.ppy.sh/b/967569)
+  - [Various Artists - Chordial Blitz (DourGent) [Venetian Drums - Sheenoboo]](https://osu.ppy.sh/b/1703314)
+  - [Joywave - Yolomania Selections (Daybreak) [1.1x]](https://osu.ppy.sh/b/2066048)
+  - [BilliumMoto - life flashes before weeb eyes (Hydria) [Tech]](https://osu.ppy.sh/b/2041466)
+  -	[void - Just Hold On (LovelySerenade) [Flawless Diversion (challenge)]](https://osu.ppy.sh/b/1467814)
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:206]:
```suggestion
- [void - Just Hold On (LovelySerenade) [Flawless Diversion (challenge)]](https://osu.ppy.sh/b/1467814)
```
I think you pressed tab twice

File: wiki/Tournaments/MCNC/2019/en.md [hunk 18]
```diff
@@ -0,0 +1,354 @@
...
+  - **[Kano - Stella-rium (DJ Noriken Bootleg) (Lude) [Koneko's Celestial // 4K]](https://osu.ppy.sh/b/1253374)**
+
+
+--------------------------------
+
+## Match results
+
+### Finals
+
+| Blue team | | | Red team | Match link |
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:219]: Any dates?
And use this format:

`| Player 1 |  |  | Player 2 | Match link |`

Apply for all matches

File: wiki/Tournaments/MCNC/2019/en.md [hunk 19]
```diff
@@ -0,0 +1,354 @@
...
+| **- xiaoluoli -**  | **5** | 0 |  Sakaki | -Win by Default- |
+| Lo_liFish2000  | 0 | **5** |  **AradORiOn_MMM** | [#1](https://osu.ppy.sh/community/matches/52845212) |
+| LFWY  | 0 | **5** |  **AradORiOn_MMM** | -Win by Default- |
+| **[RT]Mrqianmo**  | **5** | 2 |  [Crz]I-Night | [#1](https://osu.ppy.sh/community/matches/52846150) |
+| Namida KL  | 0 | **5** |  **[Crz]Satori** | -Win by Default- |
+| Namida KL  | 0 | **5** |  **ImmortalphoeniX** | -Win by Default- |
+| ying huo  | 0 | **5** |  **Stink God** | -Win by Default- |
+| ying huo  | 0 | **5** |  **Pureforever** | -Win by Default- |
+
+---------------------------------
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:295]:
```suggestion

```
Delete LB

File: wiki/Tournaments/MCNC/2019/en.md [hunk 20]
```diff
@@ -0,0 +1,354 @@
...
+
+--------------------------------
+
+## Match results
+
+### Finals
+
+| Blue team | | | Red team | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **[Crz]Murasame**  | **7** | 3 |  [Crz]FolAH1217 | [#1](https://osu.ppy.sh/community/matches/54022298) |
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:221]:
```suggestion
| **\[Crz\]Murasame** | **7** | 3 | \[Crz\]FolAH1217 | [#1](https://osu.ppy.sh/community/matches/54022298) |
```
You also only need 1 cell padding for filled content.
And 2 cell padding if that's empty.
Apply for all.

File: wiki/Tournaments/MCNC/2019/en.md [hunk 21]
```diff
@@ -0,0 +1,354 @@
...
+| **[RT]Mrqianmo**  | **5** | 2 |  [Crz]I-Night | [#1](https://osu.ppy.sh/community/matches/52846150) |
+| Namida KL  | 0 | **5** |  **[Crz]Satori** | -Win by Default- |
+| Namida KL  | 0 | **5** |  **ImmortalphoeniX** | -Win by Default- |
+| ying huo  | 0 | **5** |  **Stink God** | -Win by Default- |
+| ying huo  | 0 | **5** |  **Pureforever** | -Win by Default- |
+
+---------------------------------
+
+## Ruleset
+### Tournament Rules
```
Comment (fajar13k) [inline wiki/Tournaments/MCNC/2019/en.md:298]:
```suggestion

### Tournament Rules
```
Hit enter

**Comment by xz1z1z1z1z** [comment]
> I think i overdid it, apply

Thank u very much!! and i have changed it all

File: wiki/Tournaments/MCNC/2019/en.md [hunk 22]
```diff
@@ -285,51 +285,51 @@ The osu!mania 4K Chinese National Cup 2019 was run by various community members.
...
 
-### Match Instructions
+### Match instructions
 
 1. A referee will create a multiplayer room 15 minutes in advance. Players must gather during this period.
-    - Room settings are osu!mania, Head-To-Head, Win Condition: 'ScoreV2'. Room name must be "MCNC: (Red) vs (Blue)".
+   - Room settings are osu!mania, Head-To-Head, Win Condition: 'ScoreV2'. Room name must be "MCNC: (Red) vs (Blue)".
 2. Warmups during the Group Stages may not have a length higher than 3 minutes. Bracket Stages may be no longer than 5 minutes (unless agreed by both side). Warmups are optional when allowed.
 3. Players are allowed to ban 1 map each for each match alternating between the players with no restrictions except that each player must ban from mappool.
 Before matches will start, the captain of each player will roll and the one with the highest roll will choose their warmup first (if they would like their warmup), pick maps from the mappool first and ban one map from the mappool second. Picks will alternate as well.
```
Comment (MegaApplePi) [inline wiki/Tournaments/MCNC/2019/en.md:335]: is this part of rule 3 above?

File: wiki/Tournaments/MCNC/2019/en.md [hunk 23]
```diff
@@ -285,51 +285,51 @@ The osu!mania 4K Chinese National Cup 2019 was run by various community members.
...
-    - Have more than 30 hours of playtime or 1,000,000 total hits in osu!mania.
-    - Have not been banned/majorly restricted in the past year.
-    - Pass a background check by osu!staff and the tournament staff.
-    - Include the answer "lfnb!" in the question "Do you have anything else to say?"
+   - Have more than 30 hours of playtime or 1,000,000 total hits in osu!mania.
+   - Have not been banned/majorly restricted in the past year.
+   - Pass a background check by osu!staff and the tournament staff.
+   - Include the answer "lfnb!" in the question "Do you have anything else to say?"
 3. We reserve the right to deny any and all suspicious accounts/players from participating in the tournament.
 Players must abide to the osu! terms of service.
```
Comment (MegaApplePi) [inline wiki/Tournaments/MCNC/2019/en.md:306]: is this part of rule 3 above?

Comment (xz1z1z1z1z) [reply wiki/Tournaments/MCNC/2019/en.md:306]: this is part of rule 2

Comment (cl8n) [reply wiki/Tournaments/MCNC/2019/en.md:313]: he was asking about line 306, I'm going to combine 305 and 306

**Pull request merged**